### ISSUE TITLE
Move repo to tskit-dev

### ISSUE THREAD

**Issue by awohns** [awohns/tsdate#139]
Here are some tasks to accomplish when we move the repo:

- [ ] Change formatting to [Black](https://black.readthedocs.io/en/stable/) 
- [ ] Add pre-commit hooks
- [ ] Change master branch to main
- [ ] Check continuous integration tests are harmonized

**Comment by benjeffery** [comment]
@awohns Would you like me to handle master->main? And what about pytest? Mergify? Would be good for dev experience to be same across the repos.

**Comment by awohns** [comment]
Those changes all sound good and very happy for you to handle it if you have the bandwidth!

**Comment by awohns** [comment]
All set on this @benjeffery ?

**Comment by benjeffery** [comment]
@awohns I've edited the top todo list to the current status - will pick this up again next week I think.

**Comment by hyanwong** [comment]
You can also add `switch to pytests` and tick it off, if you want to feel like you've done more without, erm, actually doing any more.

**Comment by awohns** [comment]
How do we feel about a 1.0 release before the rest of these todos are done?

**Comment by hyanwong** [comment]
Is there any reason to do so? Is this for the preprint? I suspect it will take another week for the tsinfer stuff (metadata + mismatch) to be sorted ready for release, anyway?

**Comment by awohns** [comment]
Yes, for the preprint. But I suppose we don't need to release this at same time as preprint

**Comment by hyanwong** [comment]
Do we want to wait for tsinfer 0.2.0 for the preprint?

**Comment by awohns** [comment]
I don't think we need to necessarily

**Comment by hyanwong** [comment]
Do we not want to release a version with the mismatch parameters documented?

**Comment by benjeffery** [comment]
Let me know if you are going to release as I'll get the wheel building stuff done beforehand.
